Content.Server/Ghost/Roles/Components/GhostRoleMobSpawnerComponent.cs

```diff
@@ -10,17 +10,23 @@ namespace Content.Server.Ghost.Roles.Components
     [Access(typeof(GhostRoleSystem))]
     public sealed partial class GhostRoleMobSpawnerComponent : Component
     {
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("deleteOnSpawn")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("deleteOnSpawn")]
```

Comment:
```suggestion
        [DataField]
```
Using DataField gives the field ReadWrite VV automatically, and it will infer the name in camel case from the field name.

These still exist the old way in a bunch of classes because they haven't been updated to the newer style. But the new style should be used for new stuff (and it wouldn't hurt to clean up the others in this file while you're here).